CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Added the ability to check default `KubeletConfig` configuration. Compliance Operator
   will fetch `KubeletConfig` for each node and save it to `/kubeletconfig/role/{{role}}`.
 
+- `ComplianceScan` custom resources now contain conditions, similar to `ComplianceSuite`
+  conditions that were implemented earlier. This aids in debugging of scheduled scans.
```

Comment:
Do we have a version number to use in place of earlier?

Reply:
I don't know how precise my git archeology was, but I put a number into the changelog.